packages/core/upload/admin/src/components/AssetDialog/BrowseStep/index.tsx

```diff
@@ -100,8 +100,10 @@ export interface FileWithType extends File {
   type?: string;
 }
 
+type AllowedTypes = 'files' | 'images' | 'videos' | 'audios';
```

Comment:
You shouldn't have to define this type in multiple places (it's also in AssetCard), export/import it instead